### DISCUSSION TITLE
Segmentation Fault Error during Supermuc seissol run at  Initialize Friction Model step

### DISCUSSION THREAD

**Discussion by jwjeremy** [SeisSol/SeisSol#742] [Q&A] [answered]
**Describe the bug:**
Segmentation fault error encountered with running seissol in supermuc. The stout shows the segmentation error and the sterr is empty about the error. Please see the attached stout file.

The error is probably due to the mesh as previous mesh files work well with the parameter files and the rest of the settings. 

**Expected behavior:**
To be able to initialize the fault interface

**To Reproduce:**
The seissol setup and mesh files are in /hppfs/work/pn49ha/di75yik/TOHOKU_FL33
The Mesh can also be seen here. 
https://drive.google.com/drive/folders/1B0VY-Hh00t7bIJwGJN7zJ_Dd2Lv2IXdA?usp=sharing

[2461877.Tohoku_docker_test1.err.txt](https://github.com/SeisSol/SeisSol/files/10094820/2461877.Tohoku_docker_test1.err.txt)
[2461877.Tohoku_docker_test1.out.txt](https://github.com/SeisSol/SeisSol/files/10094821/2461877.Tohoku_docker_test1.out.txt)

**Comment by Thomas-Ulrich** [comment]
I think you are running out of memory.
32e6 cells might be too large for 16 nodes.

**Reply by jwjeremy** [reply]
Thanks for the advice! I created a model with a smaller mesh size: 8029290 cells, and increased the number of nodes to 128. However the segmentation error was still raised during the fault initialization. 
[2474357.Tohoku_docker_test1.err.txt](https://github.com/SeisSol/SeisSol/files/10107032/2474357.Tohoku_docker_test1.err.txt)

I updated the mesh file in /hppfs/work/pn49ha/di75yik/TOHOKU_FL33 and the stout and sterr of the job are attached. 
[2474357.Tohoku_docker_test1.out.txt](https://github.com/SeisSol/SeisSol/files/10107034/2474357.Tohoku_docker_test1.out.txt)


Many thanks!!!

**Comment by Thomas-Ulrich** [comment, marked as answer]
Hi,
you should update your seissol to latest commit
(git pull)
as a similar problem was fixed by:
https://github.com/SeisSol/SeisSol/pull/715

**Reply by jwjeremy** [reply]
Thanks! The segmentation fault is still raised with updated seissol. The branch and commit id are Master  and #00bbf63 .

**Reply by Thomas-Ulrich** [reply]
can you update the permission on /hppfs/work/pn49ha/di75yik/TOHOKU_FL33?
I can read it.

**Reply by jwjeremy** [reply]
Updated! Thanks!!

**Reply by Thomas-Ulrich** [reply]
Sebastian found the mesh had an issue, see https://github.com/SeisSol/SeisSol/issues/751

**Reply by jwjeremy** [reply]
Thank you so much!
